src/app/components/Breadcrumbs/Breadcrumbs.jsx

```diff
@@ -10,49 +11,31 @@ import { Link } from 'react-router';
  */
 const Breadcrumbs = ({ links, pageType, onClickHandler }) => {
   const homeLink = (
-    <li key="home">
-      <Link
-        to="/"
-        onClick={event => onClickHandler(event)}
-      >
-        ResearchNow
-      </Link>
-    </li>
+    <Link
+      to="/"
+      onClick={event => onClickHandler(event)}
+    >
+      ResearchNow
+    </Link>
   );
 
   const crumbTrail = () => {
     const crumbs = [homeLink];
 
     if (links && links.length && pageType !== 'home') {
-      links.forEach((link, iterator) => {
-        const linkKey = `links-${iterator}`;
-        if (iterator < links.length - 1) {
-          crumbs.push(
-            <li key={linkKey}>
-              <Link to={link.href}>{link.text}</Link>
-            </li>,
-          );
-        } else {
-          crumbs.push(<li key={linkKey}>{link.text}</li>);
-        }
+      links.forEach((link) => {
+        crumbs.push(
+          <Link to={link.href}>{link.text}</Link>,
+        );
       });
     }
 
     return crumbs;
   };
 
   const crumbs = crumbTrail();
-
   return (
-    <div className="grid-row">
-      <nav
-        aria-label="Breadcrumbs"
-        className="grid-col-10 sfr-center nypl-breadcrumbs"
-      >
-        <span className="nypl-screenreader-only">You are here:</span>
-        <ol>{crumbs}</ol>
-      </nav>
-    </div>
+    <DS.Breadcrumb breadcrumbs={crumbs} />
```

Comment:
I'm a bit torn on this naming - On one hand, I think it's better to allow products to wrap DS components using their own functionality, shown in this Breadcrumbs class.  But it does mean that we want to namespace DS components.

Reply:
I absolutely think this is the way to go, it makes it clear where this component is coming from and it is important to make a distinction between the design components and the local components implementing them.